app/views/internal/feedback_messages/index.html.erb

```diff
@@ -80,7 +80,7 @@
     <div id="vomitReactionsBodyContainer" class="collapse hide" aria-labelledby="vomitReactionsHeader" data-parent="#vomitReactions">
       <div class="card-body" style="overflow: scroll; max-height: 500px;">
         <% @vomits.each do |reaction| %>
-          <% next unless reaction.reactable.published %>
+          <% next if reaction.reactable_type == "Article" && !reaction.reactable.published %>
```

Comment:
Question, does this mean we won't ever see comment vomit feedback messages?

Reply:
Comments show up normally!

I don't love this for a few reasons (syntax, checking a type), but the issue was initially that the Comment model doesn't have a published method (since we don't have draft comments). I think for the time being this is fine.

Reply:
oh right bc a comment is not an article and not and its `next if` Sorry, the next got me, I was thinking do this if 🤦‍♀

Reply:
Yeah `next if...` isn't my favorite pattern ever, but for the short-term it should help Michael out to not see unpublished article vomits, so :man_shrugging: